docs/ardana/testing.md

```diff
@@ -210,6 +210,91 @@ prepare_infra
 ...
 ```
 
+## Custom Ardana Gerrit CI builds
+
+The default integration test job launched by the Gerrit Jenkins CI uses
+a `standard` generated input model, with an integrated deployer, 2
```

Comment:
A 'standard' flavour model traditionally would be a standalone deployer, vs 'dac' jobs which use 'integrated' deployer...

So maybe the definition of 'standard' has changed over time, but would be good to eliminate any sources of confusion/ambiguity.

Reply:
Looking below it appears that the "standard" jobs are using standalone rather than the integrated deployer, no?

Reply:
Yes, that is correct. I corrected the mistake.